src/components/ServiceCard/index.jsx

```diff
@@ -0,0 +1,50 @@
+import Icon from "../Icon";
+import PropTypes from "prop-types";
+import { icons } from "../Icon";
+import { useNavigate } from "react-router-dom";
+
+const ServiceCard = ({
+  id = "0",
+  name = "Service Provider",
+  role = "Role",
+  location = "Location",
+  icon = "user",
+  availableDays = ["S", "M", "T", "W", "T", "F", "S"],
+}) => {
+  const navigate = useNavigate();
+  const onClick = () => navigate(`/provider/${id}`);
+  return (
+    <div
+      onClick={onClick}
+      className={`flex border-2 hover:cursor-pointer border-primary-700 rounded-lg bg-[url(assets/images/footballField.svg)] bg-cover bg-no-repeat relative`}
+    >
+      <span className="w-3/5 h-full block bg-primary-700 left-[105px] absolute opacity-80 rounded-tl-[60px] rounded-tr-[220px] rounded-br-[32px]	rounded-bl-[85px]"></span>
+      <div className="flex z-10 gap-10 h-40 p-4 max-w-lg items-center">
+        <Icon icon={icon} containerType="square" alt="icon" />
```

Comment:
Remove invalid and unnecessary `alt` prop for this component.

It have itself `alt` prop with legend `icon`